src/index.js

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
```

Comment:
Can we have a single index.js file? i don't think there's any need for multiple files as such. :)

Reply:
Fixed